### PR TITLE
big update on future courses

### PR DESCRIPTION
**Pull request by eglerean**
Added courses, schedules and some registration links.

Still work in progress here and there, but it is good to make it live now.

### CODE REVIEW

File: training/scip/matlab-advanced.rst [hunk 1]
```diff
@@ -1,39 +1,36 @@
...
 
 Part of `Scientific Computing in Practice <https://scicomp.aalto.fi/training/scip/index.html>`__ lecture series at Aalto University.
 
-**Audience:** Aalto employees and students.
+**Audience:** Everyone with basic knowledge of Matlab
 
 **About the course:**
 
-MATLAB is a high-level technical computing language and interactive environment for algorithm development, data visualization, data analysis and numerical computation in general.  — The Mathworks Inc.
+MATLAB is a high-level technical computing lan§guage and interactive environment for algorithm development, data visualization, data analysis and numerical computation in general.  — The Mathworks Inc.
```
Comment (rkdarst) [inline training/scip/matlab-advanced.rst:11]:
```suggestion
MATLAB is a high-level technical computing language and interactive environment for algorithm development, data visualization, data analysis and numerical computation in general.  — The Mathworks Inc.
```

File: training/scip/summer-kickstart.rst [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ==================================
-Jun 2020 / FGCI Summer Kickstart
+June 2020 / FGCI Summer Kickstart
```
Comment (rkdarst) [inline training/scip/summer-kickstart.rst:2]:
```suggestion
June 2021 / FGCI Summer Kickstart
```
Can you verify these are the dates for 2021?

**Review by rkdarst — APPROVED** [review]
two fixes I think it should happen.

File: training/scip/summer-kickstart.rst (hunk 2 above)
Comment (eglerean) [reply training/scip/summer-kickstart.rst:2]: thanks for the fixes. The dates for summer kickstat are updated for 2021 assuming the course will happen on the same week as last summer.

**Review by wmvanvliet — APPROVED** [review]
That was all from me.

**Pull request merged**